### PR TITLE
Add Nyx chain to registry 

### PR DESCRIPTION
**Pull request by mfahampshire**
Adding initial information about the Nyx Blockchain, used by the Nym mixnet. Let me know if you need any other information from us.

### CODE REVIEW

File: nyx/chain.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nyx",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://nymtech.net/",
+  "pretty_name": "Nym",
+  "chain_id": "nyx",
+  "bech32_prefix": "n",
```
Comment (JeremyParish69) [inline nyx/chain.json:9]: please register this prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: nyx/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            "denom": "NYM",
+            "exponent": 6
+          }
+        ],
+        "base": "unym",
+        "name": "Nym",
+        "display": "nym",
+        "symbol": "NYM",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/",
```
Comment (JeremyParish69) [inline nyx/assetlist.json:22]: can we please expand this urls to point the image you're uploading?

File: nyx/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          "svg": "https://raw.githubusercontent.com/"
```
Comment (JeremyParish69) [inline nyx/assetlist.json:23]: please remove since no svg is being added

File: nyx/chain.json [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+      {
+        "address":"https://rpc.nyx.ezstaking.io/", 
+        "provider":"ezstaking"
+      }, 
+      {
+        "address":"https://rpc.nym.forbole.com/", 
+        "provider":"forbole"
+      }, 
+      {
+        "address":"https://rpc-nyx.testnet.run/",
```
Comment (JeremyParish69) [inline nyx/chain.json:138]: is this one just for a testnet? we have a testnets folder for that, but shouldn't be included here if it's not for mainnet

File: nyx/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply nyx/assetlist.json:22]: https://raw.githubusercontent.com/cosmos/chain-registry/master/nyx/images/nyx.png

File: nyx/chain.json (hunk 4 above)
Comment (mfahampshire) [reply nyx/chain.json:138]: No, its a mainnet validator called 'testnet run', which i understand is a little confusing. Its for mainnet though.

File: nyx/assetlist.json (hunk 3 above)
Comment (mfahampshire) [reply nyx/assetlist.json]: removed

File: nyx/assetlist.json (hunk 2 above)
Comment (mfahampshire) [reply nyx/assetlist.json]: updated, apologies for missing that the first time

File: nyx/chain.json (hunk 1 above)
Comment (mfahampshire) [reply nyx/chain.json:9]: PR is here, just waiting on it to be merged. https://github.com/satoshilabs/slips/pull/1480

Comment (mfahampshire) [reply nyx/chain.json:9]: update: this is now merged

**Pull request merged**